<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHomeScreen</class>
 <widget class="QWidget" name="CHomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="pixmap">
      <pixmap resource="image.qrc">:/image/aip.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>817</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="titleCn">
     <property name="text">
      <string>电机综合测试仪</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleEn">
     <property name="text">
      <string>Motor comprehesive tester</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>95</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="styleSheet">
         <string notr="true">color: green</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOnline">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>在线列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSystem">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>系统设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfig">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>型号管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnData">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>数据分析</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>进入测试</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>退出程序</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>60</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
